android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -63,7 +63,7 @@ class PlayerModule(context: ReactApplicationContext) : BitmovinBaseModule(contex
         promise: Promise,
     ) = promise.unit.resolveOnUiThread {
         if (players.containsKey(nativeId)) {
-            throw IllegalArgumentException("Duplicate player creation for id $nativeId")
+            return@resolveOnUiThread // key can be reused to access the same native instance (see NativeInstanceConfig)
```

Comment:
If the new TS instance has different configurations, they are ignored (which is fine IMO).
Should we check the configs here and show a warning if they do not match?

Reply:
Added a warning: https://github.com/bitmovin/bitmovin-player-react-native/pull/456/commits/6619a6ef28c496ebf4ed620019c7076c5e24a51d

Unfortunately, it is not a player event as JS is not yet listening for events when the player is create (only when it is attached to a view).

Reply:
That seems like a simple but effective solution 😄 👍